sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFnTester.java

```diff
@@ -508,6 +517,20 @@ public void onTimer(TimerInternals.TimerData timer) throws Exception {
     return MoreObjects.firstNonNull(elems, Collections.<WindowedValue<T>>emptyList());
   }
 
+  @SuppressWarnings({"unchecked", "rawtypes"})
+  public <T> List<WindowedValue<T>> getOrCreateOutput(TupleTag<T> tag) {
```

Comment:
Renamed this one and getOutput to respectively getMutableOutput and getImmutableOutput, and fixed a couple more small things in DoFnTester that followed.